src/View/Helper/FormHelper.php

```diff
@@ -169,7 +167,17 @@ public function input($fieldName, array $options = [])
                 if ($options['inline']) {
                     $options['label'] = $this->injectClasses($options['type'] . '-inline', (array)$options['label']);
                 }
-
+                break;
+            case 'radio':
+                $isInline = (isset($options['inline']) && $options['inline'] === true);
+                if (!$isInline || $this->_align === 'horizontal') {
+                    $this->templater()->remove('radioFormGroup');
+                }
+                if (!$isInline) {
+                    $this->templater()->add([
+                        'nestingLabel' => '<div class="radio">' . $this->templater()->get('nestingLabel') . '</div>'
```

Comment:
> Besides the wrapper is forced by bootstrap so I think there is no reason for the user to change the template.

Sorry I don't buy that argument. The whole point of the template system is to allow people the ability to customize.